### DISCUSSION TITLE
purchase a modified dgt pi w?

### DISCUSSION THREAD

**Discussion by chessfanatic4314** [EdNekebno/DGTCentaurMods#144] [Q&A]
can anyone sell me a modified dgt pi w or pi 2 w ? looking to get my dad a gift hes a chess master

thanks

**Comment by wormstein** [comment]
I don't have one but let us know where you are based, maybe someone can offer an option to get a w/w2 off the market and ask a skilled guy to solder the header.

**Reply by chessfanatic4314** [reply]
im in arizona but can provide a shipping label if someone is even out of country  - even happy to mail it to whoever i can get one with a return label and pay for it
